modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -8,6 +8,8 @@
   let(:claim_id) { '600131328' }
   let(:all_claims_path) { "/services/claims/v2/veterans/#{veteran_id}/claims" }
   let(:claim_by_id_path) { "/services/claims/v2/veterans/#{veteran_id}/claims/#{claim_id}" }
+  let(:claim_id_with_items) { '600236068' }
+  let(:claim_by_id_with_items_path) { "/services/claims/v2/veterans/#{veteran_id}/claims/#{claim_id_with_items}" }
```

Comment:
Can these 2 `let` statements be moved down into the context of your spec?
Keeping these vars closer to the spec(s) using them will make it much easier to mentally navigate the specs in the future.